internal/controller/server/execution/execution.go

```diff
@@ -202,6 +205,34 @@ func (e *execution) ExecuteJob(ctx context.Context, executionData *clientCPproto
 	return jobID, err
 }
 
+func (e *execution) GetJob(ctx context.Context, start *executorCPproto.ExecutorID) (*executorCPproto.Job, error) {
+	jobID, clientJob, err := e.scheduler.FetchJob(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	metadataName := clientJob.JobName
+	metadata, err := e.metadataRepo.GetValue(ctx, metadataName)
+	if err != nil {
+		return nil, err
+	}
+	imageName := metadata.ImageName
+
+	job := &executorCPproto.Job{
+		HasJob:    "yes",
```

Comment:
change HasJob to boolean type, yes/no in a string form is not good

Reply:
There were some issue with using false when zero values got assumed on nil response. Did handled it now so changing back to boolean.